docs/gc.md

```diff
@@ -146,6 +158,22 @@ There are a number of advantages to using arraylets.
 
 [`-Xgcpolicy:nogc`](xgcpolicy.md#nogc) handles only memory allocation and heap expansion, but doesn't reclaim any memory. The GC impact on runtime performance is therefore minimized, but if the available Java heap becomes exhausted, an `OutOfMemoryError` exception is triggered and the VM stops.
 
+#### When to use
+
+This policy is not suited to the majority of Java applications. However, the following use cases apply:
+
+- Testing during development
+
+    - GC performance: Use `nogc` as a baseline when testing the performance of other GC policies, including the provision of a low-latency baseline.
+
+    - Application memory: Use `nogc` to test your settings for allocated memory. If you use `-Xmx` to set the heap size that should not be exceeded, your application will crash with a heap dump if it tries to exceed your memory limit.
```

Comment:
consider using 'terminate' instead of 'crash'

Reply:
Updated to:

"Application memory: Use `nogc` to test your settings for allocated memory. If you use `-Xmx` to set the heap size that should not be exceeded, your application terminates with a heap dump if it tries to exceed your memory limit."